Installation.md

```diff
@@ -8,13 +8,25 @@ To install Swift for TensorFlow, download one of the packages below and follow t
 
 ## Releases
 
+| Download | Version | Date | 
+|----------|---------|------|
+| [Xcode 10](https://storage.googleapis.com/swift-tensorflow-artifacts/versions/v0.3/rc1/swift-tensorflow-RELEASE-0.3-osx.pkg) | v0.3 | April 23, 2019 |
+| [Ubuntu 18.04 (CPU Only)](https://storage.googleapis.com/swift-tensorflow-artifacts/versions/v0.3/rc1/swift-tensorflow-RELEASE-0.3-ubuntu18.04.tar.gz) | v0.3 April 23, 2019 |
+| [Ubuntu 18.04 (CUDA 10.0)](https://storage.googleapis.com/swift-tensorflow-artifacts/versions/v0.3/rc1/swift-tensorflow-RELEASE-0.3-cuda10.0-cudnn7-ubuntu18.04.tar.gz) | v0.3 | April 23, 2019 |
+| [Ubuntu 18.04 (CUDA 9.2)](https://storage.googleapis.com/swift-tensorflow-artifacts/versions/v0.3/rc1/swift-tensorflow-RELEASE-0.3-cuda9.2-cudnn7-ubuntu18.04.tar.gz) | v0.3 | April 23, 2019 |
+
+<details>
+  <summary>Older Releases</summary>
```

Comment:
How about "Older Packages"? This would make it consistent with the "Development Snapshots" section.